Overview.src.html

```diff
@@ -3965,9 +3962,63 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 is itself associated with <span title=concept-Response-response>response</span>'s
 <span title=concept-response-header-list>header list</span>.
 
-<p>A <code>Response</code> object's <span title=concept-Body-body>body</span> is its
-<span title=concept-Response-response>response</span>'s
-<span title=concept-response-body>body</span>.
+<p>A <code>Response</code> object also has an associated
+<dfn title=concept-Response-readable-stream>readable stream</dfn> of type
+<span title=concept-ReadableStream>ReadableStream</span> which is initially null.
+
+<p class = "note no-backref">The type of the associated
+<span title=concept-Response-readable-stream>readable stream</span> is defined as
+<span title=concept-ReadableStream>ReadableStream</span>, but it is currently discussed. Depending
+on the discussion, the type might be changed to <code>ReadableByteStream</code>. Hence the type name
+is not strictly specified at this moment, though the behavior is specified because
+<code>ReadableByteStream</code> has all methods and properties which
+<span title=concept-ReadableStream>ReadableStream</span> has.
+
+<p>A <code>Response</code> object also has an associated
```

Comment:
A better phrasing would be "A Response object is said to be locked if its associated readable stream..." Similarly for disturbed.

Although maybe these concepts should apply to abstract "responses" and not to Response objects? I am not sure.